lib/StackAnalysis/StackAnalysis.cpp

```diff
@@ -51,6 +51,12 @@ static opt<std::string> StackAnalysisOutputPath("stack-analysis-output",
                                                 value_desc("path"),
                                                 cat(MainCategory));
 
+static opt<bool> UseFunctionSymbols("use-function-symbols",
+                                    desc("Use function symbols in CFEP "
+                                         "election"),
+                                    cat(MainCategory),
+                                    init(false));
```

Comment:
We already use function symbols. Call this `UseFunctionSymbolsOnly`/`use-function-symbols-only`.